spring-native-image/Dockerfiles/Dockerfile.jvm

```diff
@@ -1,5 +1,5 @@
 # Using Oracle GraalVM for JDK 17
```

Comment:
@sachin-pikle maybe we should update the comment too?

Reply:
Yes. Will fix it in the next update.